### DISCUSSION TITLE
Add "last login" field to the list of user details returned via the API

### DISCUSSION THREAD

**Discussion by vlad-ko** [codecov/feedback#242] [Ideas]
**What product do you want to improve?**
Codecov API 

**Is your feature request related to a problem? Please describe.**
This is a request coming from one of our customers. 

**Describe the solution you'd like**
This endpoint:
https://docs.codecov.com/reference/users_list
should return another field in the resulting payload, which would contain a "last_login" field along with other fields returned by this API call. 

**Describe alternatives you've considered**
No other way to get this data. We could add another endpoint, but that is not necessary and might be more cumbersome for this request.

**Comment by rohan-at-sentry** [comment]
This is not currently part of the roadmap. Converting to a discussion to gauge broader interest from the community
